pkg/rpc/client/client.go

```diff
@@ -2,13 +2,14 @@ package client
 
 import (
 	"bytes"
-	log "github.com/Sirupsen/logrus"
-	"github.com/docker/infrakit/pkg/spi"
-	"github.com/gorilla/rpc/v2/json2"
 	"net"
 	"net/http"
 	"net/http/httputil"
 	"sync"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/infrakit/pkg/spi"
+	"github.com/gorilla/rpc/v2/json2"
```

Comment:
Revert?  If we're going to churn on import convention, i'd very much prefer we have a tool mandate the convention.